### PR TITLE
Alacritty version 0.14.1-rc1

### PR DESCRIPTION
**Pull request by chrisduerr**
---

Just added all the minor patches that have built up since the last release, should be a pretty uncontroversial one I think. Features held back for 0.15.0, alacritty_terminal is the only changed dependency crate and it's non-breaking.

### CODE REVIEW

**Review by kchibisov — CHANGES_REQUESTED** [review]
I'd wait for winit to release 0.30.6.

https://github.com/rust-windowing/winit/pull/4042

**Comment by chrisduerr** [comment]
Will also pull in the non-breaking dep bump from master.

File: alacritty_terminal/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ sections should follow the order `Added`, `Changed`, `Deprecated`, `Fixed` and
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.24.2
+
+- Semantic escape search across wide characters
```
Comment (kchibisov) [inline alacritty_terminal/CHANGELOG.md:13]: it doesn't have section.

File: CHANGELOG.md [hunk 2]
```diff
@@ -51,6 +63,7 @@ Notable changes to the `alacritty_terminal` crate are documented in its
 - Invalid URL highlights after terminal scrolling
 - Hollow block cursor not spanning multiple chars being edited inside the IME preview
 - Vi inline search only working for direct key input without modifiers
+- Crash when pressing certain modifier keys on macOS 15+
```
Comment (kchibisov) [inline CHANGELOG.md:66]: wrong rebase.

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,6 +41,7 @@ Notable changes to the `alacritty_terminal` crate are documented in its
 - Moved config option `shell` to `terminal.shell`
 - `ctrl+shift+u` binding to open links to `ctrl+shift+o` to avoid collisions with IMEs
 - Use `Beam` cursor for single char cursor inside the IME preview
+- Always emit `1` for the first parameter when having modifiers in kitty keyboard protocol
```
Comment (kchibisov) [inline CHANGELOG.md:44]: wrong rebase.

**Review by kchibisov — CHANGES_REQUESTED** [review]

**Review by kchibisov — APPROVED** [review]

**Pull request merged**